app/controllers/api/v1/reservations_controller.rb

```diff
@@ -0,0 +1,12 @@
+class ReservationsController < ApplicationController
+  def index
```

Comment:
Great Job @Dachrono including the controller in Version 1 of the API. In order of the controller, to have the correct access to the Application Controller, it should be called using the corresponding folder path. In this case, should follow the following naming: `class Api::V1::ReservationsController < ApplicationController`

Reply:
Tky for the comments, Solved